purchases/src/test/java/com/revenuecat/purchases/PostingTransactionsTests.kt

```diff
@@ -171,30 +173,19 @@ class PostingTransactionsTests {
             onError = { _, _ -> }
         )
         assertThat(postedProductInfoSlot.isCaptured).isTrue
-        assertThat(postedProductInfoSlot.captured.duration).isEqualTo(expectedSubscriptionPeriod)
-        assertThat(postedProductInfoSlot.captured.introDuration).isEqualTo(expectedIntroPricePeriod)
-        assertThat(postedProductInfoSlot.captured.trialDuration).isEqualTo(expectedFreeTrialPeriod)
-        verify(exactly = 1) {
```

Comment:
this `verify` was called in a lot of unit tests where it didn't feel necessary

Reply:
Hmm I guess it makes sure we are only posting to the backend once... I think it's ok to remove it, but we probably want to have a test to verify that we don't post multiple times.

Reply:
good point, i just think one test should test one thing...but let me add one for that case!

Reply:
okay, actually, testing it's call just once happens for any of the tests where we validate certain parameters going through to backend, so i think that's covered.